misc/requirements.txt

```diff
@@ -1,8 +0,0 @@
-# packages needed for test and release
```

Comment:
there are other scripts in this directory that use this file for documenting dependencies (e.g. release.py). accounting for this removal is straight-forward on our end, but then I feel like where losing something here and moving version documentation elsewhere.

Reply:
Do you think it would be better to leave that in `misc/` and just move the E2E tests to their own top level directory? That way the existing utilities in `misc/` are unaffected by changes made to the e2e tests.

Reply:
that sounds fine with me.